packages/board/src/vaadin-board.js

```diff
@@ -1,7 +1,12 @@
 /**
  * @license
- * Copyright (c) 2017 - 2022 Vaadin Ltd.
- * This program is available under Commercial Vaadin Developer License 4.0, available at https://vaadin.com/license/cvdl-4.0.
+ * Copyright (C) 2000-2022 Vaadin Ltd
```

Comment:
Should we align these with core components? e.g. like this:

```suggestion
 * Copyright (c) 2000 - 2022 Vaadin Ltd.
```

Reply:
Done